### PR TITLE
Blog changes

### PR DESCRIPTION
**Pull request by UtsavSomaiya**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -123,7 +123,8 @@
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <title>Retail Shop</title>
-        <link rel="stylesheet" href="https://unpkg.com/tailwindcss@2.0.2/dist/tailwind.min.css">
+        <!-- https://unpkg.com/tailwindcss@2.0.2/dist/tailwind.min.css -->
```
Comment (hirenkeraliya) [inline index.php:126]: why?

**Review by hirenkeraliya — CHANGES_REQUESTED** [review]

File: index.php (hunk 1 above)
Comment (UtsavSomaiya) [reply index.php:126]: Yesterday I am on your PC but net is not available than download this package

File: cart_template.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+                <input class="mx-2 border text-center w-8 input-quantity" type="number">
+                <button class="px-3 py-1 rounded-md bg-gray-300 increase">+</button>
+            </div>
+            <div class="flex pl-3 font-semibold text-lg w-16 text-center" >
+                <div class="currency-sign"></div>
+                <div class="price"></div>
+            </div>
+            <span class="px-3 py-1 rounded-md bg-red-300 text-white">x</span>
+        </div>
+</template>
```
Comment (hirenkeraliya) [inline cart_template.php:18]: indentation?

File: cart_template.php [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+<template id="cart-hidden">
+    <input type="hidden" name="productId[]" class="product-id">
+    <input type="hidden" name="productQuantity[]"class="product-quantity">
+    <input type="hidden" name="discount_id" class="discount-id">
+    <input type="hidden" name="discount_tier_id" class="discount-tier-id">
+</template>
```
Comment (hirenkeraliya) [inline cart_template.php:25]: End of line?

File: custom.js [hunk 4]
```diff
@@ -16,18 +16,21 @@ function addToCart(id) {
             var index = cart.findIndex((obj => obj.id == id));
             if (document.getElementById('stock-' + id).innerHTML > cart[index].quantity) {
                 cart[index].quantity = parseInt(cart[index].quantity) + 1;
+            } else {
+                alert('Stock is over!!');
```
Comment (hirenkeraliya) [inline custom.js:20]: Out of stock

**Review by hirenkeraliya — COMMENTED** [review]
discount.js is still using the createElement

**Comment by UtsavSomaiya** [comment]
> discount.js is still using the createElement

This change is done in next PR.

**Review by hirenkeraliya — APPROVED** [review]

**Pull request merged**